shared/unattended/RHEL-7-series.ks

```diff
@@ -49,6 +49,9 @@ prelink
 %end
 
 %post
+# FIXME: Remove the /dev/ttysclp0 workaround when s390x console bug is resolved
+# https://bugzilla.redhat.com/show_bug.cgi?id=1351968
+function ECHO { for TTY in `[ -e /dev/ttysclp0 ] && echo ttysclp0; cat /proc/consoles | cut -f1 -d' '`; do echo "$*" > /dev/$TTY; done }
 function ECHO { for TTY in `cat /proc/consoles | cut -f1 -d' '`; do echo "$*" > /dev/$TTY; done }
```

Comment:
You forgot to remove this implementation of `ECHO`.

Reply:
Oups, true...